_partials/_cloudwatch-data-exporter.md

```diff
@@ -22,11 +22,19 @@
 
     <Procedure>
 
-    Create an IAM role following this [AWS blog post][cross-account-iam-roles].
+    Create an IAM OIDC provider in your AWS account. If the IAM console is used, it should look like:
 
-    When you create the IAM OIDC provider:
-    - Set the URL to the [region where the exporter is being created][reference].
```

Comment:
the reference link that should be somewhere in this document should be deleted too, shouldn't it?

Reply:
good catch, removed.